### ISSUE TITLE
The image messages from realsense camera stop publishing after some time.

### ISSUE THREAD

**Issue by arieleo2690** [AutoModelCar/AutoModelCarWiki#11]
I have the next issue in AutoModelCar version 2:
All image messages including /rgb /ir and /depth topics stop to publishing after some time working, however, the realsense camera is still on. Other topics as /scan keep publishing messages correctly.
By other hand the timestamp of published images is not changing in time and the header.stamp value is the same for topics like /app/camera/rgb/image_color and /image_raw.

Regards.

**Comment by AutoModelCar** [comment]
Did you patch the car? Follow the structure and change the wire. 
https://github.com/AutoModelCar/AutoModelCarWiki/wiki/Patch-for-Model-Car-Version-2-(stabilize-power-supply)

**Comment by arieleo2690** [comment]
Yes, I've made the patch process, the image messages stop publishing yet after some time. I'm using ethernet connection.

**Comment by AutoModelCar** [comment]
Do you have power supply for odroid (5v, 4A). Could you test it with power supply. If it is stable with power supply, the Problem is your battery or your patching. The input voltage of odroid should not be under 4.85 volt.

**Comment by arieleo2690** [comment]
I fixed the patch and the camera doesn't shut down anymore. However, the timestamp for the images is the same always, I'm trying to solve it in the realsense_camera package in source code src/sr300_nodelet.cpp and the function 'ros::Time SR300Nodelet::getTimestamp()' always returns the same value for ros::Time::now()

Do you have any idea?

**Comment by Inveduc** [comment]
Hi, where can I buy a automodelcar wiki?

**Comment by AutoModelCar** [comment]
@Inveduc  https://github.com/AutoModelCar/AutoModelCarWiki/wiki

**Comment by Inveduc** [comment]
Thank you, good day
